setup/create_db.pl

```diff
@@ -221,6 +221,7 @@ sub create_mysqltables {
 	on_piped bool NOT NULL default '0',
 	on_spamassassin bool NOT NULL default '0',
 	on_vacation bool NOT NULL default '0',
+	spam_drop bool NOT NULL default '0',
```

Comment:
Can you add this field to the commented out block in `create_postgrestables` (see Line 330)?